<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>710</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>710</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Media Cross-Coder</string>
  </property>
  <property name="statusTip">
   <string>Legyen egy jó napod</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetMinAndMaxSize</enum>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="MainLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="TopFrame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QSplitter" name="splitter">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhNone</set>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="handleWidth">
            <number>10</number>
           </property>
           <property name="childrenCollapsible">
            <bool>true</bool>
           </property>
           <widget class="QFrame" name="TopLeftFrame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="leftMargin">
              <number>2</number>
             </property>
             <property name="topMargin">
              <number>2</number>
             </property>
             <property name="rightMargin">
              <number>2</number>
             </property>
             <property name="bottomMargin">
              <number>2</number>
             </property>
             <item>
              <widget class="QTreeView" name="projectView">
               <property name="frameShape">
                <enum>QFrame::WinPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="iconSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
               <property name="textElideMode">
                <enum>Qt::ElideRight</enum>
               </property>
               <property name="autoExpandDelay">
                <number>2</number>
               </property>
               <property name="uniformRowHeights">
                <bool>true</bool>
               </property>
               <property name="animated">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="ControllBoxLayout">
               <property name="spacing">
                <number>10</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>5</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>5</number>
               </property>
               <item>
                <layout class="QVBoxLayout" name="TopControllBoxLayout">
                 <property name="spacing">
                  <number>5</number>
                 </property>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout">
                 <property name="spacing">
                  <number>5</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="buttonNewProject">
                   <property name="text">
                    <string>Új terv</string>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>16</width>
                     <height>24</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="buttonAddMedia">
                   <property name="text">
                    <string>Hozzáadása</string>
                   </property>
                   <property name="icon">
                    <iconset>
                     <normalon>:/icons/Images/media_add.png</normalon>
                    </iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>16</width>
                     <height>24</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="buttonAppendMedia">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>Összefűzés</string>
                   </property>
                   <property name="icon">
                    <iconset>
                     <normalon>:/icons/Images/media_app.png</normalon>
                    </iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>16</width>
                     <height>24</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="buttonDeleteMedia">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>Tőrlés</string>
                   </property>
                   <property name="icon">
                    <iconset>
                     <normalon>:/icons/Images/media_del.png</normalon>
                    </iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>16</width>
                     <height>24</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="BottomContollBoxLayout">
                 <property name="spacing">
                  <number>5</number>
                 </property>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QFrame" name="TopRightFrame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>4</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <property name="leftMargin">
              <number>2</number>
             </property>
             <property name="topMargin">
              <number>2</number>
             </property>
             <property name="rightMargin">
              <number>2</number>
             </property>
             <property name="bottomMargin">
              <number>2</number>
             </property>
             <item>
              <widget class="QTextBrowser" name="textBrowser">
               <property name="acceptDrops">
                <bool>false</bool>
               </property>
               <property name="autoFillBackground">
                <bool>true</bool>
               </property>
               <property name="inputMethodHints">
                <set>Qt::ImhMultiLine|Qt::ImhSensitiveData</set>
               </property>
               <property name="frameShape">
                <enum>QFrame::WinPanel</enum>
               </property>
               <property name="documentTitle">
                <string notr="true"/>
               </property>
               <property name="lineWrapMode">
                <enum>QTextEdit::WidgetWidth</enum>
               </property>
               <property name="html">
                <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="overwriteMode">
                <bool>true</bool>
               </property>
               <property name="tabStopWidth">
                <number>180</number>
               </property>
               <property name="acceptRichText">
                <bool>true</bool>
               </property>
               <property name="cursorWidth">
                <number>1</number>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
               </property>
               <property name="placeholderText">
                <string notr="true"/>
               </property>
               <property name="openExternalLinks">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBarTotal">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBarCurrent">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>250</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>250</height>
         </size>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddMedia"/>
   <addaction name="actionAppendMedia"/>
   <addaction name="actionMediaDelete"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;Fájl</string>
    </property>
    <addaction name="actionAddProject"/>
    <addaction name="actionAddMedia"/>
    <addaction name="actionAppendMedia"/>
    <addaction name="actionMediaDelete"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Súgó</string>
    </property>
    <addaction name="actionQTAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="mcc.qrc">
     <normaloff>:/icons/Images/cross-script.png</normaloff>
     <normalon>:/new/prefix1/Images/select.png</normalon>:/icons/Images/cross-script.png</iconset>
   </property>
   <property name="text">
    <string>Kilépés</string>
   </property>
   <property name="statusTip">
    <string>Kilépés az alkalmazásból</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionAddMedia">
   <property name="icon">
    <iconset resource="mcc.qrc">
     <normaloff>:/icons/Images/media_add.png</normaloff>:/icons/Images/media_add.png</iconset>
   </property>
   <property name="text">
    <string>Média hozzáadása</string>
   </property>
   <property name="toolTip">
    <string>Media hozzáadása</string>
   </property>
   <property name="statusTip">
    <string>Média hozzáadása a tervhez.</string>
   </property>
  </action>
  <action name="actionAddProject">
   <property name="text">
    <string>Új terv</string>
   </property>
   <property name="toolTip">
    <string>Új terv létrehozása</string>
   </property>
  </action>
  <action name="actionAppendMedia">
   <property name="icon">
    <iconset resource="mcc.qrc">
     <normaloff>:/icons/Images/media_app.png</normaloff>:/icons/Images/media_app.png</iconset>
   </property>
   <property name="text">
    <string>Média hozzáfűzése</string>
   </property>
   <property name="toolTip">
    <string>Media hozzáfűzése</string>
   </property>
   <property name="statusTip">
    <string>Média hozzáfűzése az aktív médiához.</string>
   </property>
  </action>
  <action name="actionMediaDelete">
   <property name="icon">
    <iconset resource="mcc.qrc">
     <normaloff>:/icons/Images/media_del.png</normaloff>:/icons/Images/media_del.png</iconset>
   </property>
   <property name="text">
    <string>Média törlése</string>
   </property>
   <property name="toolTip">
    <string>Média tőrlése</string>
   </property>
   <property name="statusTip">
    <string>Média eltávolítása a tervből.</string>
   </property>
  </action>
  <action name="actionQTAbout">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>QT Névjegy ...</string>
   </property>
   <property name="statusTip">
    <string>QT Keretrendszer névjegye.</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="mcc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>579</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>MediaOpen()</slot>
 </slots>
</ui>
